### PR TITLE
[9.0][MIG] event_registration_mass_mailing module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Updated views

cc @Tecnativa

### CODE REVIEW

**Review by luismontalba — APPROVED** [review]
Tested on runbot

File: event_registration_mass_mailing/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Mass mailing from events
+========================
+
+This module adds actions to send mass mailings and allows to add to mailing
+lists from event registrations tree view.
+
+Usage
+=====
+
+* Add 'Action > Mass mailing to partners' button in the partner views: with this
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: s/partner view/registrations view

File: event_registration_mass_mailing/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  button, you can send an email to the selected registrations.
+
+* Add 'Action > Add to mailing list' button in the partner views: with this
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: s/partner view/registrations view

File: event_registration_mass_mailing/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Mass mailing from events",
+    'category': 'Marketing',
+    'version': '9.0.1.1.0',
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/__openerp__.py]: Why not 9.0.1.0.0?

File: event_registration_mass_mailing/models/email_template.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+class EmailTemplate(models.Model):
+    _inherit = 'mail.template'
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form',
+                        toolbar=False, submenu=False):
+        res = super(EmailTemplate, self).fields_view_get(
+            view_id=view_id, view_type=view_type,
+            toolbar=toolbar, submenu=submenu)
+        if 'arch' in res:
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/models/email_template.py]: You can save one indentation making:
```
if 'arch' not in res:
    return res
doc = ...

File: event_registration_mass_mailing/models/email_template.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            doc = etree.XML(res['arch'])
+            for node in doc.xpath("//field[@name='model_id']"):
+                domain = node.get('domain')
+                if domain:
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/models/email_template.py]: The same:
```
if not domain:
    continue
domain = ...
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by chienandalu — APPROVED** [review]
tested on runbot

**Review by pedrobaeza — APPROVED** [review]

**Comment by rafaelbn** [comment]
Please @cubells review comments. Functionally module is OK 👍

**Pull request merged**